com.kristianduske.TrenchBroom.json

```diff
@@ -0,0 +1,144 @@
+{
+    "app-id": "com.kristianduske.TrenchBroom",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "trenchbroom.sh",
+    "rename-desktop-file": "trenchbroom.desktop",
+    "rename-icon": "trenchbroom",
+    "finish-args": [
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--filesystem=host",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "*.la",
+        "*.a"
+    ],
+    "build-options": {
```

Comment:
Not needed.

Default flags for **Freedesktop 18.08**:
```
$ flatpak run --command=cat org.freedesktop.Sdk//18.08 /etc/flatpak-builder/defaults.json
{
  'libdir': '/usr/lib/x86_64-linux-gnu',
  'cflags': '-march=x86-64 -mtune=generic -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection',
  'cxxflags': '-march=x86-64 -mtune=generic -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection',
  'cppflags': '',
  'ldflags': '-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed'
}
```

Reply:
Ah, I see. I cleared the cflags and cxxflags.